pkg/printers/internalversion/describe_test.go

```diff
@@ -1313,3 +1314,85 @@ func TestPrintLabelsMultiline(t *testing.T) {
 		}
 	}
 }
+
+func TestDescribeUnstructuredContent(t *testing.T) {
+	testCases := []struct {
+		expected   string
+		unexpected string
+	}{
+		{
+			expected: "API Version:\tv1\n",
+		},
+		{
+			expected: "\nMetadata:\n",
+		},
+		{
+			expected: "\n  Name:\tMyName\n",
+		},
+		{
+			expected: "\n  Creation Timestamp:\t2017-04-01T00:00:00Z\n",
+		},
+		{
+			expected: "\nItems:\n  Item Bool:\ttrue\n  Item Int:\t42\n",
+		},
+		{
+			expected: "\nURL:\thttp://localhost\n",
+		},
+		{
+			unexpected: "\nDummy 1:\tpresent\n",
+		},
+		{
+			unexpected: "Dummy 3",
+		},
+		{
+			unexpected: "Dummy3",
+		},
+		{
+			unexpected: "dummy3",
+		},
+		{
+			unexpected: "dummy 3",
+		},
+	}
+	out := new(bytes.Buffer)
+	w := NewPrefixWriter(out)
+	obj := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "v1",
+			"kind":       "Test",
+			"dummy1":     "present",
+			"dummy2":     "present",
+			"metadata": map[string]interface{}{
+				"name":              "MyName",
+				"namespace":         "MyNamespace",
+				"creationTimestamp": "2017-04-01T00:00:00Z",
+				"resourceVersion":   123,
+				"uid":               "00000000-0000-0000-0000-000000000001",
+				"dummy3":            "present",
+			},
+			"items": []interface{}{
+				map[string]interface{}{
+					"itemBool": true,
+					"itemInt":  42,
+				},
+			},
+			"url":    "http://localhost",
+			"status": "ok",
+		},
+	}
+	printUnstructuredContent(w, LEVEL_0, obj.UnstructuredContent(), "", ".dummy1", ".metadata.dummy3")
+	output := out.String()
+
+	for _, test := range testCases {
```

Comment:
Now that the order is consistent, we should make sure we test the ordering.  Doing so should simplify this check.